dipy/align/cpd.py

```diff
@@ -184,10 +187,13 @@ class DeformableRegistration:
         Number of eigenvectors to use in lowrank calculation.
     """
 
+    @warning_for_keywords()
     def __init__(
         self,
         X,
         Y,
+        args=(),
+        *,
```

Comment:
it does not look correct. read and try this: https://stackoverflow.com/questions/64423083/why-cant-both-args-and-keyword-only-arguments-be-mixed-with-args-and-kwargs

Reply:
got it. corrected.

Reply:
I think your decorator does not work with that. I tried:

```python
@warning_for_keywords(from_version="1.0.0", until_version="10.0.0")
def func_with_kwonly_args(a, b, *args, c=3):
      return a + b + c


func_with_kwonly_args(1, 2, 4)
```

and it crashes. Can you look into it and add a test in a new PR.  Thanks

Reply:
Ah ok, after multiple tests, you do not need the decorator when you have `
*args`. Which is a good news !